docs/developer_guide.md

```diff
@@ -259,56 +269,67 @@ Debugging options are available. Click on the required option :
 <details><summary>Use gdb to debug in Docker</summary>
 
 1. Build a project in a debug mode :
-	```
+	```bash
 	make docker_build BAZEL_BUILD_TYPE=dbg
 	```
 
 2. Run the container :
-	```
+	```bash
 	docker run -it --cap-add=SYS_PTRACE --security-opt seccomp=unconfined -v ${PWD}:/ovms -p 9178:9178 --entrypoint bash openvino/model_server-build:latest
 	```
-3.	Assuming resnet50 model is prepared for OVMS in /models catalog recompile the OpenVINO Model Server with debug symbols using command:
-    ```
-	[root@72dc3b874772 ovms]# bazel build //src:ovms -c dbg
-    [root@72dc3b874772 ovms]# gdb --args ./bazel-bin/src/ovms --model_name resnet --model_path /models
+3.	Prepare resnet50 model for OVMS in /models catalog and recompile the OpenVINO Model Server in docker container with debug symbols using command:
+	```bash
+	mkdir -p /models/1 && wget -P /models/1 https://storage.openvinotoolkit.org/repositories/open_model_zoo/2022.1/models_bin/2/resnet50-binary-0001/FP32-INT1/resnet50-binary-0001.bin && wget -P /models/1 https://storage.openvinotoolkit.org/repositories/open_model_zoo/2022.1/models_bin/2/resnet50-binary-0001/FP32-INT1/resnet50-binary-0001.xml
+	```
+    ```bash
```

Comment:
The indentation in this line does not seem right

Reply:
fixed